CI: documentation build failing?

### Describe your issue.

Looks like we're getting the same documentation build failure in gh-21164, gh-21167, gh-21194, etc.

### Reproducing Code Example

```python
NA
```


### Error message
<details>

```shell
# for testing
# @echo installed scipy ac893f4 matches git version ac893f4; exit 1
mkdir -p source/_contents
for file in source/tutorial/stats/*.md; do \
	basename=$(basename $file .md); \
	output_name="source/_contents/${basename}.ipynb"; \
	/home/circleci/.pyenv/versions/3.11.9/bin/python -m jupytext --output "$output_name"  $file; \
done
[jupytext] Reading source/tutorial/stats/hypothesis_bartlett.md in format md
[jupytext] Writing source/_contents/hypothesis_bartlett.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_chi2_contingency.md in format md
[jupytext] Writing source/_contents/hypothesis_chi2_contingency.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_chisquare.md in format md
[jupytext] Writing source/_contents/hypothesis_chisquare.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_dunnett.md in format md
[jupytext] Writing source/_contents/hypothesis_dunnett.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_fisher_exact.md in format md
[jupytext] Writing source/_contents/hypothesis_fisher_exact.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_fligner.md in format md
[jupytext] Writing source/_contents/hypothesis_fligner.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_friedmanchisquare.md in format md
[jupytext] Writing source/_contents/hypothesis_friedmanchisquare.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_jarque_bera.md in format md
[jupytext] Writing source/_contents/hypothesis_jarque_bera.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_kendalltau.md in format md
[jupytext] Writing source/_contents/hypothesis_kendalltau.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_kurtosistest.md in format md
[jupytext] Writing source/_contents/hypothesis_kurtosistest.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_levene.md in format md
[jupytext] Writing source/_contents/hypothesis_levene.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_normaltest.md in format md
[jupytext] Writing source/_contents/hypothesis_normaltest.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_odds_ratio.md in format md
[jupytext] Writing source/_contents/hypothesis_odds_ratio.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_shapiro.md in format md
[jupytext] Writing source/_contents/hypothesis_shapiro.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_skewtest.md in format md
[jupytext] Writing source/_contents/hypothesis_skewtest.ipynb
[jupytext] Reading source/tutorial/stats/hypothesis_spearmanr.md in format md
[jupytext] Writing source/_contents/hypothesis_spearmanr.ipynb
[jupytext] Reading source/tutorial/stats/sampling.md in format md
[jupytext] Writing source/_contents/sampling.ipynb
mkdir -p build/html build/doctrees
LANG=C /home/circleci/.pyenv/versions/3.11.9/bin/python -msphinx -WT --keep-going  -b html -d build/doctrees -j1  source build/html 
Running Sphinx v7.4.4
SciPy (VERSION pull/21164)
loading translations [en]... done
Converting `source_suffix = '.rst'` to `source_suffix = {'.rst': 'restructuredtext'}`.
[autosummary] generating autosummary for: building/blas_lapack.rst, building/compilers_and_options.rst, building/cross_compilation.rst, building/distutils_equivalents.rst, building/index.rst, building/introspecting_a_build.rst, building/redistributable_binaries.rst, building/understanding_meson.rst, dev/api-dev/api-dev-toc.rst, dev/api-dev/array_api.rst, ..., tutorial/stats/probability_distributions.rst, tutorial/stats/quasi_monte_carlo.rst, tutorial/stats/resampling.rst, tutorial/stats/sampling.md, tutorial/stats/sampling_dau.rst, tutorial/stats/sampling_dgt.rst, tutorial/stats/sampling_hinv.rst, tutorial/stats/sampling_pinv.rst, tutorial/stats/sampling_srou.rst, tutorial/stats/sampling_tdr.rst
[autosummary] generating autosummary for: /home/circleci/repo/doc/source/reference/generated/czt-function.rst, /home/circleci/repo/doc/source/reference/generated/odr-function.rst, /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.rst, /home/circleci/repo/doc/source/reference/generated/scipy.cluster.hierarchy.ClusterNode.rst, /home/circleci/repo/doc/source/reference/generated/scipy.cluster.hierarchy.DisjointSet.rst, /home/circleci/repo/doc/source/reference/generated/scipy.cluster.hierarchy.average.rst, /home/circleci/repo/doc/source/reference/generated/scipy.cluster.hierarchy.centroid.rst, /home/circleci/repo/doc/source/reference/generated/scipy.cluster.hierarchy.complete.rst, /home/circleci/repo/doc/source/reference/generated/scipy.cluster.hierarchy.cophenet.rst, /home/circleci/repo/doc/source/reference/generated/scipy.cluster.hierarchy.correspond.rst, ..., /home/circleci/repo/doc/source/reference/generated/scipy.stats.yeojohnson.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.yeojohnson_llf.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.yeojohnson_normmax.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.yeojohnson_normplot.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.yulesimon.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.zipf.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.zipfian.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.zmap.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.zscore.rst, /home/circleci/repo/doc/source/reference/generated/scipy.test.rst
[autosummary] generating autosummary for: /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.__getitem__.rst, /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.__len__.rst, /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.__mul__.rst, /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.count.rst, /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.from_cython.rst, /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.function.rst, /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.index.rst, /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.signature.rst, /home/circleci/repo/doc/source/reference/generated/scipy.LowLevelCallable.user_data.rst, /home/circleci/repo/doc/source/reference/generated/scipy.cluster.hierarchy.ClusterNode.get_count.rst, ..., /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.NumericalInversePolynomial.u_error.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.RatioUniforms.rvs.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.SimpleRatioUniforms.rvs.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.SimpleRatioUniforms.set_random_state.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.TransformedDensityRejection.hat_area.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.TransformedDensityRejection.ppf_hat.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.TransformedDensityRejection.rvs.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.TransformedDensityRejection.set_random_state.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.TransformedDensityRejection.squeeze_area.rst, /home/circleci/repo/doc/source/reference/generated/scipy.stats.sampling.TransformedDensityRejection.squeeze_hat_ratio.rst
loading intersphinx inventory 'asv' from https://asv.readthedocs.io/en/stable/objects.inv...
loading intersphinx inventory 'matplotlib' from https://matplotlib.org/stable/objects.inv...
loading intersphinx inventory 'mpmath' from https://mpmath.org/doc/current/objects.inv...
loading intersphinx inventory 'neps' from https://numpy.org/neps/objects.inv...
loading intersphinx inventory 'numpy' from https://numpy.org/doc/stable/objects.inv...
loading intersphinx inventory 'python' from https://docs.python.org/3/objects.inv...
loading intersphinx inventory 'statsmodels' from https://www.statsmodels.org/stable/objects.inv...
myst v3.0.1: MdParserConfig(commonmark_only=False, gfm_only=False, enable_extensions={'colon_fence', 'substitution', 'dollarmath'}, disable_syntax=[], all_links_external=False, links_external_new_tab=False, url_schemes=('http', 'https', 'mailto', 'ftp'), ref_domains=None, fence_as_directive=set(), number_code_blocks=[], title_to_header=False, heading_anchors=0, heading_slug_func=None, html_meta={}, footnote_transition=False, words_per_minute=200, substitutions={}, linkify_fuzzy_links=True, dmath_allow_labels=True, dmath_allow_space=True, dmath_allow_digits=True, dmath_double_inline=False, update_mathjax=False, mathjax_classes='tex2jax_process|mathjax_process|math|output_area', enable_checkboxes=False, suppress_warnings=[], highlight_code_blocks=True)
myst-nb v1.1.1: NbParserConfig(custom_formats={}, metadata_key='mystnb', cell_metadata_key='mystnb', kernel_rgx_aliases={}, eval_name_regex='^[a-zA-Z_][a-zA-Z0-9_]*$', execution_mode='auto', execution_cache_path='', execution_excludepatterns=['_contents/*.ipynb'], execution_timeout=30, execution_in_temp=False, execution_allow_errors=False, execution_raise_on_error=False, execution_show_tb=False, merge_streams=False, render_plugin='default', remove_code_source=False, remove_code_outputs=False, code_prompt_show='Show code cell {type}', code_prompt_hide='Hide code cell {type}', number_source_lines=False, output_stderr='show', render_text_lexer='myst-ansi', render_error_lexer='ipythontb', render_image_options={}, render_figure_options={}, render_markdown_format='myst', output_folder='build', append_css=True, metadata_to_fm=False)
Using jupyter-cache at: /home/circleci/repo/doc/build/.jupyter_cache
building [mo]: targets for 0 po files that are out of date
writing output... 
building [html]: targets for 373 source files that are out of date
updating environment: [new config] 4485 added, 0 changed, 0 removed
reading sources... [  0%] building/blas_lapack
...
reading sources... [100%] tutorial/stats/sampling_tdr

WARNING: Summarised items should not include the current module. Replace 'scipy.datasets.face' with 'face'.
WARNING: Summarised items should not include the current module. Replace 'scipy.integrate.quad' with 'integrate.quad'.
WARNING: Summarised items should not include the current module. Replace 'scipy.ndimage.generic_filter' with 'ndimage.generic_filter'.
WARNING: Summarised items should not include the current module. Replace 'scipy.ndimage.generic_filter1d' with 'ndimage.generic_filter1d'.
WARNING: Summarised items should not include the current module. Replace 'scipy.ndimage.geometric_transform' with 'ndimage.geometric_transform'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.linkage' with 'linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.fcluster' with 'fcluster'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.dendrogram' with 'dendrogram'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.linkage' with 'linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.fcluster' with 'fcluster'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.dendrogram' with 'dendrogram'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.linkage' with 'linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.fcluster' with 'fcluster'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.dendrogram' with 'dendrogram'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.cophenet' with 'cophenet'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.ward' with 'ward'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.fcluster' with 'fcluster'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.fcluster' with 'fcluster'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.from_mlab_linkage' with 'from_mlab_linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.single' with 'single'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.complete' with 'complete'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.ward' with 'ward'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.median' with 'median'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.is_monotonic' with 'is_monotonic'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.inconsistent' with 'inconsistent'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.is_valid_im' with 'is_valid_im'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.is_valid_linkage' with 'is_valid_linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.leaders' with 'leaders'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.leaders' with 'leaders'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.leaves_list' with 'leaves_list'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.inconsistent' with 'inconsistent'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.maxRstat' with 'maxRstat'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.maxdists' with 'maxdists'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.median' with 'median'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.maxdists' with 'maxdists'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.inconsistent' with 'inconsistent'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.maxinconsts' with 'maxinconsts'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.linkage' with 'linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.fcluster' with 'fcluster'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.dendrogram' with 'dendrogram'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.linkage' with 'linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.fcluster' with 'fcluster'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.dendrogram' with 'dendrogram'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.to_mlab_linkage' with 'to_mlab_linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.linkage' with 'linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.fcluster' with 'fcluster'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.dendrogram' with 'dendrogram'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.linkage' with 'linkage'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.fcluster' with 'fcluster'.
WARNING: Summarised items should not include the current module. Replace 'scipy.cluster.hierarchy.dendrogram' with 'dendrogram'.
WARNING: Summarised items should not include the current module. Replace 'scipy.integrate.odeint' with 'odeint'.
WARNING: Summarised items should not include the current module. Replace 'scipy.integrate.solve_ivp' with 'solve_ivp'.
WARNING: Summarised items should not include the current module. Replace 'scipy.integrate.ode' with 'ode'.
WARNING: Summarised items should not include the current module. Replace 'scipy.integrate.solve_ivp' with 'solve_ivp'.
WARNING: Summarised items should not include the current module. Replace 'scipy.interpolate.insert' with 'insert'.
WARNING: Summarised items should not include the current module. Replace 'scipy.interpolate.PchipInterpolator' with 'PchipInterpolator'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.eigh' with 'eigh'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.eigh' with 'eigh'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.eigh' with 'eigh'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.solve' with 'solve'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.lstsq' with 'lstsq'.
WARNING: Summarised items should not include the current module. Replace 'scipy.misc.ascent' with 'ascent'.
WARNING: Summarised items should not include the current module. Replace 'scipy.misc.central_diff_weights' with 'central_diff_weights'.
WARNING: Summarised items should not include the current module. Replace 'scipy.misc.derivative' with 'derivative'.
WARNING: Summarised items should not include the current module. Replace 'scipy.misc.electrocardiogram' with 'electrocardiogram'.
WARNING: Summarised items should not include the current module. Replace 'scipy.misc.face' with 'face'.
WARNING: Summarised items should not include the current module. Replace 'scipy.odr.odr' with 'odr'.
WARNING: Summarised items should not include the current module. Replace 'scipy.odr.odr' with 'odr'.
WARNING: Summarised items should not include the current module. Replace 'scipy.odr.odr' with 'odr'.
WARNING: Summarised items should not include the current module. Replace 'scipy.odr.odr' with 'odr'.
WARNING: Summarised items should not include the current module. Replace 'scipy.odr.odr' with 'odr'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.bracket' with 'bracket'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.fminbound' with 'fminbound'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.bracket' with 'bracket'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.OptimizeResult' with 'OptimizeResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.OptimizeResult' with 'OptimizeResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.least_squares' with 'least_squares'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.linprog' with 'linprog'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.basinhopping' with 'basinhopping'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.bracket' with 'bracket'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize_scalar' with 'minimize_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root_scalar' with 'root_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.linprog' with 'linprog'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.quadratic_assignment' with 'quadratic_assignment'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.sweep_poly' with 'sweep_poly'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.spectrogram' with 'spectrogram'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.get_window' with 'get_window'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.get_window' with 'get_window'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.boxcar' with 'windows.boxcar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.triang' with 'windows.triang'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.blackman' with 'windows.blackman'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.hamming' with 'windows.hamming'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.hann' with 'windows.hann'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.bartlett' with 'windows.bartlett'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.flattop' with 'windows.flattop'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.parzen' with 'windows.parzen'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.bohman' with 'windows.bohman'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.blackmanharris' with 'windows.blackmanharris'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.nuttall' with 'windows.nuttall'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.barthann' with 'windows.barthann'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.cosine' with 'windows.cosine'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.exponential' with 'windows.exponential'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.tukey' with 'windows.tukey'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.taylor' with 'windows.taylor'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.lanczos' with 'windows.lanczos'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.kaiser' with 'windows.kaiser'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.kaiser_bessel_derived' with 'windows.kaiser_bessel_derived'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.gaussian' with 'windows.gaussian'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.general_cosine' with 'windows.general_cosine'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.general_gaussian' with 'windows.general_gaussian'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.general_hamming' with 'windows.general_hamming'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.dpss' with 'windows.dpss'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.chebwin' with 'windows.chebwin'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.kaiser' with 'windows.kaiser'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.medfilt2d' with 'medfilt2d'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.get_window' with 'get_window'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.upfirdn' with 'upfirdn'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.resample' with 'resample'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.upfirdn' with 'upfirdn'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.get_window' with 'get_window'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.firwin' with 'firwin'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.sosfilt' with 'sosfilt'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.boxcar' with 'boxcar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.triang' with 'triang'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.blackman' with 'blackman'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.hamming' with 'hamming'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.hann' with 'hann'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.bartlett' with 'bartlett'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.flattop' with 'flattop'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.parzen' with 'parzen'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.bohman' with 'bohman'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.blackmanharris' with 'blackmanharris'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.nuttall' with 'nuttall'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.barthann' with 'barthann'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.cosine' with 'cosine'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.exponential' with 'exponential'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.tukey' with 'tukey'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.taylor' with 'taylor'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.lanczos' with 'lanczos'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.kaiser' with 'kaiser'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.kaiser_bessel_derived' with 'kaiser_bessel_derived'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.gaussian' with 'gaussian'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.general_cosine' with 'general_cosine'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.general_gaussian' with 'general_gaussian'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.general_hamming' with 'general_hamming'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.dpss' with 'dpss'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.chebwin' with 'chebwin'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows.kaiser' with 'kaiser'.
WARNING: Summarised items should not include the current module. Replace 'scipy.sparse.linalg.LinearOperator' with 'LinearOperator'.
WARNING: Summarised items should not include the current module. Replace 'scipy.special.betainc' with 'betainc'.
WARNING: Summarised items should not include the current module. Replace 'scipy.special.betaincinv' with 'betaincinv'.
WARNING: Summarised items should not include the current module. Replace 'scipy.special.expit' with 'expit'.
WARNING: Summarised items should not include the current module. Replace 'scipy.special.exp1' with 'exp1'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.multivariate_normal' with 'multivariate_normal'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.pearsonr' with 'pearsonr'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.EmpiricalDistributionFunction' with 'EmpiricalDistributionFunction'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.EmpiricalDistributionFunction' with 'EmpiricalDistributionFunction'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.EmpiricalDistributionFunction' with 'EmpiricalDistributionFunction'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.ConstantInputWarning' with 'ConstantInputWarning'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.FitResult' with '_result_classes.FitResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.monte_carlo_test' with 'monte_carlo_test'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.permutation_test' with 'permutation_test'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.qmc.Sobol' with 'qmc.Sobol'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.BinomTestResult' with '_result_classes.BinomTestResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.DegenerateDataWarning' with 'DegenerateDataWarning'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.qmc.Sobol' with 'qmc.Sobol'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.contingency.expected_freq' with 'contingency.expected_freq'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.power_divergence' with 'power_divergence'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.chisquare' with 'chisquare'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.chisquare' with 'chisquare'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.contingency.chi2_contingency()' with 'chi2_contingency()'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.contingency.chi2_contingency()' with 'chi2_contingency()'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.contingency.expected_freq' with 'expected_freq'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.DunnettResult' with '_result_classes.DunnettResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.CensoredData' with 'CensoredData'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.CensoredData' with 'CensoredData'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.CensoredData' with 'CensoredData'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.ECDFResult' with '_result_classes.ECDFResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.EmpiricalDistributionFunction' with '_result_classes.EmpiricalDistributionFunction'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.EmpiricalDistributionFunction' with '_result_classes.EmpiricalDistributionFunction'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.CensoredData' with 'CensoredData'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.CensoredData' with 'CensoredData'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.ConstantInputWarning' with 'ConstantInputWarning'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.kruskal' with 'kruskal'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.alexandergovern' with 'alexandergovern'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.contingency.odds_ratio' with 'contingency.odds_ratio'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.rv_continuous' with 'rv_continuous'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.rv_discrete' with 'rv_discrete'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.FitResult' with '_result_classes.FitResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.nbinom' with 'nbinom'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.trim_mean' with 'trim_mean'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.rv_continuous' with 'rv_continuous'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.fit' with 'fit'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.FitResult' with '_result_classes.FitResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.ks_1samp' with 'ks_1samp'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.norm' with 'norm'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.ks_1samp' with 'ks_1samp'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.trim_mean' with 'trim_mean'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.LogRankResult' with '_result_classes.LogRankResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.logrank' with 'logrank'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.CensoredData' with 'CensoredData'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.wilcoxon' with 'wilcoxon'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.ttest_ind' with 'ttest_ind'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.chi2_contingency' with 'chi2_contingency'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.chi2_contingency' with 'chi2_contingency'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.fisher_exact' with 'fisher_exact'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.skewtest' with 'skewtest'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.CovViaPrecision' with 'CovViaPrecision'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.f_oneway' with 'f_oneway'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.rankdata' with 'rankdata'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.permutation_test' with 'permutation_test'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.monte_carlo_test' with 'monte_carlo_test'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.PearsonRResult' with '_result_classes.PearsonRResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.bootstrap' with 'bootstrap'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.ConstantInputWarning' with 'ConstantInputWarning'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.NearConstantInputWarning' with 'NearConstantInputWarning'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.qmc.LatinHypercube' with 'LatinHypercube'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.qmc.discrepancy' with 'discrepancy'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.mannwhitneyu' with 'mannwhitneyu'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.FitError' with 'FitError'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.FitError' with 'FitError'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.bootstrap' with 'bootstrap'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.uniform' with 'uniform'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.ConstantInputWarning' with 'ConstantInputWarning'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.rankdata' with 'rankdata'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.TtestResult' with '_result_classes.TtestResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.TtestResult' with '_result_classes.TtestResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.ttest_ind' with 'ttest_ind'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.TtestResult' with '_result_classes.TtestResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats._result_classes.TukeyHSDResult' with '_result_classes.TukeyHSDResult'.
WARNING: Summarised items should not include the current module. Replace 'scipy.io.matlab' with 'matlab'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.interpolative' with 'interpolative'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.blas' with 'blas'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.lapack' with 'lapack'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.cython_blas' with 'cython_blas'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.cython_lapack' with 'cython_lapack'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.cython_optimize' with 'cython_optimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.linprog' with 'linprog'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.linprog' with 'linprog'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.linprog' with 'linprog'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.linprog' with 'linprog'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.linprog' with 'linprog'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.linprog' with 'linprog'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.fmin_l_bfgs_b' with 'fmin_l_bfgs_b'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize' with 'minimize'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize_scalar' with 'minimize_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize_scalar' with 'minimize_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.minimize_scalar' with 'minimize_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.quadratic_assignment' with 'quadratic_assignment'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.quadratic_assignment' with 'quadratic_assignment'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root' with 'root'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root_scalar' with 'root_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root_scalar' with 'root_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root_scalar' with 'root_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root_scalar' with 'root_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root_scalar' with 'root_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root_scalar' with 'root_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root_scalar' with 'root_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.optimize.root_scalar' with 'root_scalar'.
WARNING: Summarised items should not include the current module. Replace 'scipy.signal.windows' with 'windows'.
WARNING: Summarised items should not include the current module. Replace 'scipy.spatial.transform' with 'transform'.
WARNING: Summarised items should not include the current module. Replace 'scipy.special.cython_special' with 'cython_special'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.multivariate_normal' with 'multivariate_normal'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.mstats' with 'mstats'.
WARNING: Summarised items should not include the current module. Replace 'scipy.stats.contingency' with 'contingency'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.interpolative' with 'linalg.interpolative'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.block_diag' with 'linalg.block_diag'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.circulant' with 'linalg.circulant'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.companion' with 'linalg.companion'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.convolution_matrix' with 'linalg.convolution_matrix'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.dft' with 'linalg.dft'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.fiedler' with 'linalg.fiedler'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.fiedler_companion' with 'linalg.fiedler_companion'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.hadamard' with 'linalg.hadamard'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.hankel' with 'linalg.hankel'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.helmert' with 'linalg.helmert'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.hilbert' with 'linalg.hilbert'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.invhilbert' with 'linalg.invhilbert'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.leslie' with 'linalg.leslie'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.pascal' with 'linalg.pascal'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.invpascal' with 'linalg.invpascal'.
WARNING: Summarised items should not include the current module. Replace 'scipy.linalg.toeplitz' with 'linalg.toeplitz'.
WARNING: Summarised items should not include the current module. Replace 'scipy.sparse.linalg' with 'linalg'.
WARNING: Summarised items should not include the current module. Replace 'scipy.sparse.csgraph' with 'csgraph'.
WARNING: Summarised items should not include the current module. Replace 'scipy.sparse.csgraph' with 'csgraph'.
WARNING: Summarised items should not include the current module. Replace 'scipy.spatial.convex_hull_plot_2d' with 'convex_hull_plot_2d'.
WARNING: Summarised items should not include the current module. Replace 'scipy.spatial.voronoi_plot_2d' with 'voronoi_plot_2d'.
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
copying assets... 
copying static files... done
copying extra files... done
copying assets: done
writing output... [  0%] building/blas_lapack
writing output... [  0%] building/compilers_and_options
writing output... [  0%] building/cross_compilation
writing output... [  0%] building/distutils_equivalents
writing output... [  0%] building/index
writing output... [  0%] building/introspecting_a_build
writing output... [  0%] building/redistributable_binaries
writing output... [  0%] building/understanding_meson
writing output... [  0%] dev/api-dev/api-dev-toc
writing output... [  0%] dev/api-dev/array_api
writing output... [  0%] dev/api-dev/nan_policy
writing output... [  0%] dev/api-dev/special_ufuncs
writing output... [  0%] dev/conduct/code_of_conduct
writing output... [  0%] dev/conduct/report_handling_manual
writing output... [  0%] dev/contributor/adding_new
writing output... [  0%] dev/contributor/benchmarking
writing output... [  0%] dev/contributor/compiled_code
writing output... [  0%] dev/contributor/continuous_integration
writing output... [  0%] dev/contributor/contributor_toc
writing output... [  0%] dev/contributor/cython
writing output... [  0%] dev/contributor/debugging_linalg_issues
writing output... [  0%] dev/contributor/development_workflow
writing output... [  1%] dev/contributor/devpy_test
writing output... [  1%] dev/contributor/pep8
writing output... [  1%] dev/contributor/public_cython_api
writing output... [  1%] dev/contributor/rendering_documentation
writing output... [  1%] dev/contributor/reviewing_prs
writing output... [  1%] dev/contributor/using_act
writing output... [  1%] dev/core-dev/index
writing output... [  1%] dev/dev_quickstart
writing output... [  1%] dev/gitwash/configure_git
writing output... [  1%] dev/gitwash/development_setup
writing output... [  1%] dev/gitwash/dot2_dot3
writing output... [  1%] dev/gitwash/git_resources
writing output... [  1%] dev/gitwash/gitwash
writing output... [  1%] dev/gitwash/useful_git
writing output... [  1%] dev/governance
writing output... [  1%] dev/hacking
writing output... [  1%] dev/index
writing output... [  1%] dev/missing-bits
writing output... [  1%] dev/roadmap
writing output... [  1%] dev/roadmap-detailed
writing output... [  1%] dev/toolchain
writing output... [  1%] dev/triage
writing output... [  1%] index

Traceback (most recent call last):
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/events.py", line 98, in emit
    results.append(listener.handler(self.app, *args))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/pydata_sphinx_theme/__init__.py", line 189, in update_and_remove_templates
    context[section] = utils._update_and_remove_templates(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/pydata_sphinx_theme/utils.py", line 139, in _update_and_remove_templates
    rendered = app.builder.templates.render(template, ctx)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/jinja2glue.py", line 200, in render
    return self.environment.get_template(template).render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/jinja2/environment.py", line 1013, in get_template
    return self._load_template(name, globals)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/jinja2/environment.py", line 972, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/jinja2glue.py", line 222, in get_source
    raise TemplateNotFound(msg)
jinja2.exceptions.TemplateNotFound: 's.html' not found in ['/home/circleci/repo/doc/source/_templates', '/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/pydata_sphinx_theme/theme/pydata_sphinx_theme/components', '/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/pydata_sphinx_theme/theme/pydata_sphinx_theme', '/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/themes/basic']

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/cmd/build.py", line 337, in build_main
    app.build(args.force_all, args.filenames)
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/application.py", line 378, in build
    self.builder.build_update()
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/builders/__init__.py", line 297, in build_update
    self.build(to_build,
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/builders/__init__.py", line 367, in build
    self.write(docnames, list(updated_docnames), method)
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/builders/__init__.py", line 614, in write
    self._write_serial(sorted(docnames))
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/builders/__init__.py", line 624, in _write_serial
    self.write_doc(docname, doctree)
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/builders/html/__init__.py", line 666, in write_doc
    self.handle_page(docname, ctx, event_arg=doctree)
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/builders/html/__init__.py", line 1101, in handle_page
    newtmpl = self.app.emit_firstresult('html-page-context', pagename,
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/application.py", line 515, in emit_firstresult
    return self.events.emit_firstresult(event, *args,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/events.py", line 124, in emit_firstresult
    for result in self.emit(name, *args, allowed_exceptions=allowed_exceptions):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/events.py", line 109, in emit
    raise ExtensionError(
sphinx.errors.ExtensionError: Handler <function update_and_remove_templates at 0x7fad10c52a20> for event 'html-page-context' threw an exception (exception: 's.html' not found in ['/home/circleci/repo/doc/source/_templates', '/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/pydata_sphinx_theme/theme/pydata_sphinx_theme/components', '/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/pydata_sphinx_theme/theme/pydata_sphinx_theme', '/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/themes/basic'])

Extension error (pydata_sphinx_theme):
Handler <function update_and_remove_templates at 0x7fad10c52a20> for event 'html-page-context' threw an exception (exception: 's.html' not found in ['/home/circleci/repo/doc/source/_templates', '/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/pydata_sphinx_theme/theme/pydata_sphinx_theme/components', '/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/pydata_sphinx_theme/theme/pydata_sphinx_theme', '/home/circleci/.pyenv/versions/3.11.9/lib/python3.11/site-packages/sphinx/themes/basic'])
make: *** [Makefile:114: html-build] Error 2
Skipping build```

</details>

### SciPy/NumPy/Python version and system information

```shell
NA
```

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -195,7 +195,7 @@ html_logo = '_static/logo.svg'
 html_favicon = '_static/favicon.ico'
 
 html_sidebars = {
-    "index": "search-button-field",
+    "index": ["search-button-field"],
     "**": ["search-button-field", "sidebar-nav-bs"]
 }
 

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -184,6 +184,10 @@ warnings.filterwarnings(
     message=r'There is no current event loop',
     category=DeprecationWarning,
 )
+# See https://github.com/sphinx-doc/sphinx/issues/12589
+suppress_warnings = [
+    'autosummary.import_cycle',
+]
 
 # -----------------------------------------------------------------------------
 # HTML output

